newsroom/wire/views.py

```diff
@@ -130,6 +130,15 @@ def delete_dashboard_caches():
         app.cache.delete(f"{HOME_ITEMS_CACHE_KEY}{company['_id']}")
 
 
+def get_products_by_card(cards, user):
```

Comment:
I already applied that condition, however it fails because we need products to activate the more news button.
However, if the user has not assigned any company or product, we did not receive any product in the UI , we got `[]` products and hence the button is not enabled. 
so, if a person has admin privileges but no companies assigned to them, we provide card products instead of company-specific products.

Reply:
and currently we did not send the product form backend if user has not product assigned, so I create this function which is send products based on card config.

Reply:
I wonder why we need that product object if we only use the id which we store on the card object anyhow